llvm/unittests/Analysis/InlineOrderPlugin/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ if (NOT WIN32 AND NOT CYGWIN)
   # it is built before the plugin.
   add_dependencies(InlineOrderPlugin intrinsics_gen)
   add_dependencies(AnalysisTests InlineOrderPlugin)
-  set_property(TARGET InlineOrderPlugin PROPERTY FOLDER "Tests/UnitTests/AnalysisTests")
+  set_property(TARGET InlineOrderPlugin PROPERTY FOLDER "Tests/Unittests/AnalysisTests")
```

Comment:
Personal opinion is that it should be `UnitTests` like it was before ("unit" is an adjective, "tests" is the noun, so two separate words).

Reply:
I seem having forgotten to remove this one, because the FOLDER is already set by `add_llvm_library`. The plugin is not added under WIN32 so I didn't notice. I updated this patch.

The case change was me trying to get the case consistent across projects before I moved to handling this inside `add_llvm_library`. With two folders differentiating only in case, the VS IDE (or CMake) either renames one of them to "UnitTests (2)", or throws an error when opening such a project.